src/api/posts/src/routes/posts.js

```diff
@@ -7,11 +7,35 @@ const postsUrl = process.env.POSTS_URL || '/';
 
 const posts = Router();
 
+const prepareData = async (ids, shouldExpand = false) => {
+  if (shouldExpand) {
+    const data = await Promise.all(
```

Comment:
No need for `data` or `await` here, just return the Promise directly:

```js
return Promise.all(
```

Reply:
Hi, I pushed a new commit.